src/main/java/me/ryanhamshire/GriefPrevention/DataStore.java

```diff
@@ -471,22 +474,18 @@ synchronized void addClaim(Claim newClaim, boolean writeToStorage)
         }
     }
 
-    private void addToChunkClaimMap(Claim claim)
+    private void addToChunkClaimMap(@NotNull Claim claim)
     {
         ArrayList<Long> chunkHashes = claim.getChunkHashes();
         for (Long chunkHash : chunkHashes)
         {
-            ArrayList<Claim> claimsInChunk = this.chunksToClaimsMap.get(chunkHash);
-            if (claimsInChunk == null)
-            {
-                this.chunksToClaimsMap.put(chunkHash, claimsInChunk = new ArrayList<>());
-            }
+            ArrayList<Claim> claimsInChunk = this.chunksToClaimsMap.computeIfAbsent(chunkHash, hash -> new ArrayList<>());
```

Comment:
Possibly should have been in a separate PR, but it is related to an unnecessary null check so I threw it in.